superset-frontend/src/dashboard/components/nativeFilters/FilterBar/ActionButtons/index.tsx

```diff
@@ -103,7 +105,7 @@ export const ActionButtons = ({
   );
 
   return (
-    <ActionButtonsContainer>
+    <ActionButtonsContainer data-test="filterbar-action-buttons" width={width || OPEN_FILTER_BAR_WIDTH}>
```

Comment:
```suggestion
    <ActionButtonsContainer data-test="filterbar-action-buttons" width={width}>
```

No need for `||` is default value is assigned.